src/GRA.Data/Repository/PsBranchSelectionRepository.cs

```diff
@@ -52,6 +52,14 @@ public async Task<int> GetCountByPerformerIdAsync(int performerId)
                 .Where(_ => _.ProgramId.HasValue && _.Program.PerformerId == performerId)
                 .CountAsync();
         }
+        public async Task<ICollection<PsBranchSelection>> GetByPerformerIdAsync(int performerId)
+        {
+            return await DbSet
+                .AsNoTracking()
+                .Where(_ => _.ProgramId.HasValue && _.Program.PerformerId == performerId)
+                .ProjectTo<PsBranchSelection>(_mapper.ConfigurationProvider)
+                .ToListAsync();
+        }
```

Comment:
The existing GetByPerformerIdAsync method should be able to be called without needing an overload